src/main/js/App.js

```diff
@@ -108,7 +122,13 @@ const App = () => {
     const createSurvey = () => {
         const survey = {
             name: surveyName,
-            questions,
+            questions: questions.map(q => {
+                if (q.type === 'dropdown') {
+                    // For dropdowns, split the input before sending it
```

Comment:
Ideally, we wouldn't be separating by comma because it prevents someone from inputting an option that contains a comma.
For now it is fine though.

Reply:
Yeah I do agree, a future optimization could be to change the way we do this but I think this is the cleanest UI mechanism for the time being.